packages/flutter/test/painting/image_cache_resize_test.dart

```diff
@@ -14,14 +14,13 @@ void main() {
   TestRenderingFlutterBinding();
 
   tearDown(() {
-    imageCache!
-      ..clear()
+    imageCache..clear()
```

Comment:
format nit: leave the `..clear()` on the next line?

Reply:
also elsewhere in this PR

Reply:
found it in two places